NAMESPACE

```diff
@@ -58,6 +58,8 @@ export(.Last.updated)
 export(fcoalesce)
 export(substitute2)
 export(DT)  # mtcars |> DT(i,j,by)  #4872
+export(fdroplevels)
+export(droplevels)
```

Comment:
as `droplevels` is in base and generic, all we need, iiuc, is the `S3method` line below you've got already. Putting `export(droplevels)` here as well will generate a warning on loading the package that `droplevels` is masked? But I'm writing from memory so I might have that wrong.

Reply:
I see and did not notice before because no warning was generated. According to [Writing R Extensions](https://stuff.mit.edu/afs/sipb/project/r-project/arch/i386_rhel3/lib/R/doc/manual/R-exts.html#Registering%20S3%20methods) putting `export(droplevels)` is not necessary but possible.

Example:
> `S3method(print, foo)`
> Since the generic print is defined in `base` it does not need to be imported explicitly.

I removed it now.